projects/packages/paypal-payments/src/paypal-ncps/class-paypal-ncps.php

```diff
@@ -0,0 +1,164 @@
+
+		if ( empty( $button_type ) || empty( $code_body ) ) {
+			return;
+		}
+
+		if ( 'stacked' === $button_type && ! empty( $code_head ) ) {
+			if ( preg_match( '/src="(.+)"/', $code_head, $matches ) ) {
+				$script_url = esc_url( $matches[1] );
+				if ( ! empty( $script_url ) ) {
+					wp_enqueue_script( 'paypal-ncps-block-head', $script_url, array(), self::PACKAGE_VERSION, false );
+					add_filter(
+						'script_loader_tag',
+						function ( $tag, $handle, $src ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+							if ( 'paypal-ncps-block-head' === $handle ) {
+								if ( false === strpos( $tag, 'data-paypal-partner-attribution-id' ) ) {
+									$tag = preg_replace( '/(\s+)src=([\'"])/', '$1 data-paypal-partner-attribution-id="WooNCPS_Ecom_Wordpress" src=$2', $tag );
+								}
+							}
+							return $tag;
+						},
+						10,
+						3
+					);
+				}
+			}
+		}
+
+		if ( 'single' === $button_type ) {
+			$code_body = str_replace( 'inline-grid', 'grid', $code_body );
+			$attr_code = 'at_code=WooNCPS_Ecom_Wordpress';
+			if ( preg_match( '/\s+action=[\'"]([^\'"]+)[\'"]/', $code_body, $matches ) ) {
+				$action = esc_url( $matches[1] );
+				if ( ! empty( $action ) && false === strpos( $action, $attr_code ) ) {
+					$parsed_url_parts   = wp_parse_url( $action );
+					$prepared_attr_code = isset( $parsed_url_parts['query'] ) ? '&' . $attr_code : '?' . $attr_code;
+					$action             = $action . $prepared_attr_code;
+					$code_body          = preg_replace( '/(\s+action=[\'"])[^\'"]+([\'"])/', '$1' . $action . '$2', $code_body );
+				}
+			}
+		}
+
+		$allow_html = 'single' === $button_type ? array(
+			'style'   => array(),
+			'form'    => array(
+				'action' => array(),
+				'method' => array(),
+				'style'  => array(),
+			),
+			'input'   => array(
+				'class' => array(),
+				'type'  => array(),
+				'value' => array(),
+			),
+			'img'     => array(
+				'src'   => array(),
+				'alt'   => array(),
+				'style' => array(),
+			),
+			'section' => array(
+				'style' => array(),
+			),
+		) : array(
+			'div'    => array(
+				'id' => array(),
+			),
+			'script' => array(),
+		);
+
+		return wp_kses( $code_body, $allow_html );
+	}
+
+	/**
+	 * Load editor styles for the block.
+	 * These are loaded via enqueue_block_assets to ensure proper loading in the editor iframe context.
+	 */
+	public static function load_editor_styles() {
+		$handle = 'jp-paypal-payments-ncps-blocks';
+
+		Assets::register_script(
+			$handle,
+			'../../dist/paypal-ncps-block/editor.js',
+			__FILE__,
+			array(
+				'css_path'   => '../../dist/paypal-ncps-block/editor.css',
+				'textdomain' => 'jetpack-paypal-payments',
+			)
+		);
+		wp_enqueue_style( $handle );
+	}
+
+	/**
+	 * Loads scripts
+	 */
+	public static function load_editor_scripts() {
+		Assets::register_script(
+			'jp-paypal-payments-ncps-blocks',
+			'../../dist/paypal-ncps-block/editor.js',
```

Comment:
You'll need to update the file name here, since we no longer use `block`:

```suggestion
			'../../dist/paypal-ncps/editor.js',
			__FILE__,
			array(
				'css_path'   => '../../dist/paypal-ncps/editor.css',
				'textdomain' => 'jetpack-paypal-payments',
			)
		);
		wp_enqueue_style( $handle );
	}

	/**
	 * Loads scripts
	 */
	public static function load_editor_scripts() {
		Assets::register_script(
			'jp-paypal-payments-ncps-blocks',
			'../../dist/paypal-ncps/editor.js',
```

Reply:
I missed these on my initial pass. Updated in [7bda0bf](https://github.com/Automattic/jetpack/pull/43932/commits/7bda0bf912ceb91116639f4a4f56a98c100f9810).